docs/topics/tour/kotlin-tour-hello-world.md

```diff
@@ -0,0 +1,115 @@
+        <img src="icon-6-todo.svg" width="20" alt="Sixth step" /> <a href="kotlin-tour-classes-part-1.md">Classes</a><br />
+        <img src="icon-7-todo.svg" width="20" alt="Final step" /> <a href="kotlin-tour-null-safety.md">Null safety</a></p>
+</microformat>
+
+Below is a simple program that prints "Hello, world!":
+
+```kotlin
+fun main() {
+    println("Hello, world!")
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="hello-world-kotlin"}
+
+In Kotlin:
+* `fun` is used to declare a function
+* the entry point is the `main` function
+* function arguments are written within parentheses `()`
+* the body of a function is written within curly braces `{}`
+* [`println()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/println.html) and [`print()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/print.html) functions print their arguments to standard output
+
+Functions will be discussed in more detail in a couple of chapters. Until then, all examples use the `main` function.
+
+## Variables
+
+All programs need to be able to store data, and variables help you to do just that. In Kotlin, you can declare:
+* read-only variables with `val`
+* mutable variables with `var`
+
+To assign a value, use the assignment operator `=`.
+
+```kotlin
+fun main() { 
+//sampleStart
+    val popcorn = 5
+    val hotdog = 7
+    var customers = 10
+    
+    //Some customers leave the queue
+    customers = 8
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="variables-kotlin"}
+
+> Variables can be declared at top level as well as within functions.
+{type="info"}
+
+As `customers` is a mutable variable, its value can be reassigned after declaration.
+
+> We recommend that you declare all variables as read-only (`val`) by default. Declare mutable variables (`var`) only if 
+> necessary.
+{type="info"}
+
+## String templates
+
+It's useful to know how to print the contents of variables to standard output. You can do this with string templates. 
+You can use template expressions to access data stored in variables and other objects, and convert them into strings.
+A string value is a sequence of characters in double quotes (`"`). Template expressions always start with a dollar sign (`$`).
+
+To evaluate a piece of code in a template expression, place the code within curly braces `{}` after the dollar sign (`$`).
+
+For example:
+
+```kotlin
+fun main() { 
+//sampleStart
+    val customers = 10
+    println("There are $customers customers")
+    //There are 10 customers
+    
+    println("There are ${customers + 1} customers")
+    //There are 11 customers
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="string-templates-kotlin"}
+
+For more information, see [String templates](strings.md).
+
+You'll notice that there aren't any types declared for variables. Kotlin has inferred the type itself: `Int`. This tour
```

Comment:
Just an idea - perhaps it's worth showing how to declare the type of a variable explicitly

Reply:
There is an example of this in the final code sample of the Basic types chapter. Is this OK?